### ISSUE TITLE
Last Message on room

### ISSUE THREAD

**Issue by geekgonecrazy** [RocketChat/Rocket.Chat.ReactNative#211]
Show last message on room.

Full message object of the last sent message is saved on the room object under lastMessage property

**Comment by geekgonecrazy** [comment]
Just to poke around on this to see what is up I tried to implement this.

First of all Modifying the room schema:
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/realm.js#L51

Added:
```
lastMessage: 'messages'
```

Then realized the `database.objects('rooms')` is completely empty... So I tried to add that.
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/realm.js#L51

Tried adding below:
```
database.write(() => {
	data.forEach(subscription =>
	   database.create('subscriptions', subscription, true));
});
```

So I figured at this point they should be available.  So I added a prop to roomItem that called a local function and attempted to do something like:
```
getLastMessage = (subscription) => {
		const rooms = database.objects('rooms').sorted('_updatedAt', true).slice();
		console.log('ROOMS', rooms);
                const room =  rooms.find(({ _id }) => _id === subscription.rid);
		return room.lastMessage;
	}
```

Seemed like this would work... but i'm probably missing something basic.
